storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst

```diff
@@ -5,9 +5,7 @@ Encrypted InnoDB tablespace backups
 ===================================
 
 InnoDB supports `data encryption for InnoDB tables
-<http://dev.mysql.com/doc/refman/8.0/en/innodb-tablespace-encryption.html>`_
-stored in file-per-table tablespaces. This feature provides an at-rest encryption
-for physical tablespace data files.
+<https://dev.mysql.com/doc/mysql-enterprise-backup/8.0/en/meb-encrypted-innodb.html>`__ stored in file-per-table tablespaces. This feature provides an at-rest encryption for physical tablespace data files.
```

Comment:
seems like pointing wrong. may be https://dev.mysql.com/doc/refman/8.0/en/innodb-data-encryption.html

Reply:
fixed